bindgen/Cargo.toml

```diff
@@ -28,7 +28,7 @@ path = "lib.rs"
 annotate-snippets = { version = "0.9.1", features = ["color"], optional = true }
 bitflags = "2.2.1"
 cexpr = "0.6"
-clang-sys = { version = "1", features = ["clang_6_0"] }
+clang-sys = { version = "1", features = ["clang_11_0"] }
```

Comment:
does this somehow breaks our clang 9 support?

Reply:
No, we have tests for this. It would if we were to call the new functions unconditionally. But CXType_Atomic was introduced alongside the new functions.